lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -89,11 +89,13 @@ If you need any further assistance then you can also [request help here](${confi
   prBody = prBody.replace('{{PRLIST}}\n', getPrList(config, branches));
   // istanbul ignore if
   if (config.prHeader) {
-    prBody = (config.prHeader || '') + '\n\n' + prBody;
+    const prHeader = String(config.prHeader || '');
+    prBody = `${prHeader}\n\n${prBody}`;
```

Comment:
should be fixed later, as `prHeader` and `prFooter` should be string.

Reply:
@viceice can you clarify?

Reply:
they should be defined on the type as string, then the `String(...)` wrapper shouldn't needed